ckanext/harvest/logic/action/get.py

```diff
@@ -130,7 +130,9 @@ def harvest_source_list(context, data_dict):
 
     sources = _get_sources_for_user(context, data_dict)
 
-    return [harvest_source_dictize(source, context) for source in sources]
+    last_job_status = data_dict.get('return_last_job_status', False)
```

Comment:
You might want to use `toolkit.asbool`, otherwise passing `return_last_job_status=False` as a GET parameter will become a string and be interpreted as `True`